packages/sanity/src/core/create/createDocumentUrls.ts

```diff
@@ -0,0 +1,31 @@
+import {type CreateLinkMetadata} from './types'
+
+// @ts-expect-error: __SANITY_STAGING__ is a global env variable set by the vite config
+const isStaging = typeof __SANITY_STAGING__ !== 'undefined' && __SANITY_STAGING__ === true
+
+function getCreateBaseUrl(customHost?: string) {
+  const host = (customHost ?? isStaging) ? 'create-staging.sanity.build' : 'www.sanity.io'
+  return `https://${host}/app/create`
+}
+
+export function getCreateLinkUrl(args: {
```

Comment:
For a second I got confused 😄 
I thought that this was creating a Link, but it's "Create" the name of the Tool / App. No change needed but I see this confusion as being somewhat common place moving forward 😂

Reply:
For the record, i tried to use `SanityCreate` in code that is exported out of the `create` directory.